ravenframework/Samplers/LimitSurfaceSearch.py

```diff
@@ -204,6 +204,10 @@ def localInputAndChecks(self,xmlNode, paramInput):
       @ In, paramInput, InputData.ParameterInput, the parsed parameters
       @ Out, None
     """
+    # check input variables to avoid <variable name='x1,x2,...'> node
+    for val in list(self.toBeSampled.keys()) + list(self.dependentSample.keys()):
+      if len(val.split(',')) > 1:
+        self.raiseAnError(IOError, f'Variables {val} defined using <variable> node can not be accepted, only single variable can be processed!')
```

Comment:
i think this should not be an error, we can just pass the same value to all the variables that part of this set, no?